javascript-bank/account.js

```diff
@@ -0,0 +1,59 @@
+/* exported Account */
+
+function Account(number, holder){
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.number = function () {
+  return this.number;
+}
+
+Account.prototype.holder = function () {
+  return this.holder;
+}
```

Comment:
> Only the constructor versions of the `type` and `amount` properties of a `Transaction` instance are usable, because JavaScript will stop looking when it sees those. The `prototype` versions are effectively overwritten (or "shadowed", as we call it) by the constructor versions.
> 
> Please remove the shadowed versions of the properties.

Like with the `Transaction` properties, please remove the shadowed versions of the `Account` properties.